src/components/Header.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Header = ({currentScript}) => {
+
+
+  return (
+    <div>
+      <header>
+        <h1>Play Reading Party</h1>
+        <nav>
+            <Link to={'/'} className='button'>Home</Link>
+            <Link to={`script/${currentScript}`} className='button' style={{display: currentScript ? 'inline':'none'}}>Open Script</Link>
```

Comment:
This could be conditionally rendered, couldn't it? Rather than always including the html for it but hiding it conditionally?

Reply:
Right, thank you for the reminder of that pattern.